From bc78a382df5f64f3665fe45779941aae250f50b1 Mon Sep 17 00:00:00 2001
From: Samuli Piippo <samuli.piippo@qt.io>
Date: Mon, 17 Feb 2020 15:33:55 +0200
Subject: [PATCH] Sysrootify qmake

For cross-compilations we need to enable qmake to sysrootify
the install paths.

Change-Id: I47d210f89203e32da59a1e41fcd6ebd28e968b25
---
 cmake/QtBuild.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cmake/QtBuild.cmake b/cmake/QtBuild.cmake
index 407f0b6524..4dbbe4c3ff 100644
--- a/cmake/QtBuild.cmake
+++ b/cmake/QtBuild.cmake
@@ -3995,7 +3995,7 @@ function(qt_generate_qconfig_cpp)
     set(QT_CONFIG_STRS "")
 
     qt_add_string_to_qconfig_cpp("") # config.input.sysroot
-    qt_add_string_to_qconfig_cpp("false") # qmake_sysrootify
+    qt_add_string_to_qconfig_cpp("true") # qmake_sysrootify
     qt_add_string_to_qconfig_cpp("${INSTALL_BINDIR}") # TODO: Host-specific
     qt_add_string_to_qconfig_cpp("${INSTALL_LIBDIR}") # TODO: Host-specific
     qt_add_string_to_qconfig_cpp("${INSTALL_DATADIR}") # TODO: Host-specific
